cheroot/test/test_core.py

```diff
@@ -283,7 +279,6 @@ def test_content_length_required(test_client):
 
 
 @pytest.mark.xfail(
-    IS_CIRCLE_CI_ENV,
     reason='https://github.com/cherrypy/cheroot/issues/106',
     strict=False,  # sometimes it passes
```

Comment:
If it's failing reliably we should probably make this strict or/and change the conditional.